lib/changeset.ex

```diff
@@ -0,0 +1,162 @@
+defmodule Solicit.Changeset do
+  @moduledoc """
+  Generic functions for working with changesets that are not packaged with Ecto.
+  """
+
+  @doc """
+  A module encapsulating the translation layer between a changeset error and the JSON error shape returned in an API response.
+  """
+  @spec code_and_description(map()) :: map()
+  def code_and_description({"can't be blank", _}),
```

Comment:
Pattern matching on the description text seems kind of brittle to me. Is there a reason not to match on the `validation` key from the tuple's second member?

Reply:
That's smart. I updated this one to match on the `validation` key. I couldn't find the `validation` key for a few others so they still match to `{"blah", _}`